app/src/main/java/chat/rocket/android/authentication/login/presentation/LoginPresenter.kt

```diff
@@ -5,30 +5,8 @@ import chat.rocket.android.authentication.presentation.AuthenticationNavigator
 import chat.rocket.android.core.lifecycle.CancelStrategy
 import chat.rocket.android.helper.OauthHelper
 import chat.rocket.android.infrastructure.LocalRepository
-import chat.rocket.android.server.domain.GetAccountsInteractor
-import chat.rocket.android.server.domain.GetConnectingServerInteractor
-import chat.rocket.android.server.domain.GetSettingsInteractor
-import chat.rocket.android.server.domain.PublicSettings
-import chat.rocket.android.server.domain.SaveAccountInteractor
-import chat.rocket.android.server.domain.SaveCurrentServerInteractor
-import chat.rocket.android.server.domain.TokenRepository
-import chat.rocket.android.server.domain.casLoginUrl
-import chat.rocket.android.server.domain.favicon
-import chat.rocket.android.server.domain.gitlabUrl
-import chat.rocket.android.server.domain.isCasAuthenticationEnabled
-import chat.rocket.android.server.domain.isFacebookAuthenticationEnabled
-import chat.rocket.android.server.domain.isGithubAuthenticationEnabled
-import chat.rocket.android.server.domain.isGitlabAuthenticationEnabled
-import chat.rocket.android.server.domain.isGoogleAuthenticationEnabled
-import chat.rocket.android.server.domain.isLdapAuthenticationEnabled
-import chat.rocket.android.server.domain.isLinkedinAuthenticationEnabled
-import chat.rocket.android.server.domain.isLoginFormEnabled
-import chat.rocket.android.server.domain.isWordpressAuthenticationEnabled
-import chat.rocket.android.server.domain.isPasswordResetEnabled
-import chat.rocket.android.server.domain.isRegistrationEnabledForNewUsers
-import chat.rocket.android.server.domain.wordpressUrl
+import chat.rocket.android.server.domain.*
```

Comment:
I think you need to change your configuration to avoid `.*` imports

Reply:
👁 👍

Reply:
Using Multiple IDE Versions. I forgot to configure one of them. 👁 👍 Thanks.